docs/users_guide/ocean/test_groups/buttermilk_bay.rst

```diff
@@ -0,0 +1,138 @@
+.. _ocean_buttermilk_bay:
+
+buttermilk_bay
+==============
+
+The ``buttermilk_bay`` test group implements a realistic regional
+simulation of tidally forced circulation in a multi-bay system 
+connected by narrow channels. The computational domain is 
+`Buttermilk Bay, Massachusetts <https://tinyurl.com/4zpz36ft>`_.
+
+.. _ocean_buttermilk_bay_default:
+
+default
```

Comment:
Maybe state somewhere that the ramp feature is used?

Reply:
Good idea, will do.